lib/colors.py

```diff
@@ -70,8 +70,18 @@ def find_max_value(hist):
 
     return color_index
 
-#def find_color_range(image):
-#    """Returns array """
-#    for hist_color in image_colors(image):
+def find_color_range(image):
+    """Returns array """
+    for hist_color in image_colors(image):
+        find_max_range(hist_color)
 
-#        find_max_range(hist_color)
+def hsv_max(image):
+    hist = image_colors(image)
+    maxs = []
```

Comment:
Would we want to convert this section to a comprehension? Perhaps calling a small helper function?